packages/headless/src/features/facets/facet-search-set/specific/specific-facet-search-set-slice.test.ts

```diff
@@ -21,6 +28,10 @@ describe('FacetSearch slice', () => {
     state = getFacetSearchSetInitialState();
   });
 
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
```

Comment:
Why is this `afterEach` block needed? It seems the tests before were not depending on it?

Similar comment for the category-facet-search test file,